pkg/cluster/kubefedcluster_assets.go

```diff
@@ -0,0 +1,353 @@
+// Code generated for package cluster by go-bindata DO NOT EDIT. (@generated)
```

Comment:
ok, so here we include the generated file in the git repo because `toolchain-common` is a library for the operators. However, we should have a `make` goal to generate this file, and I can't see it in your PR. Or did I miss something? 

Alternatively, we could try to use [`go generate` comments](https://blog.golang.org/generate) to trigger the call to `go-bindata` during the operator build, and thus avoid committing this file in the repo...

Reply:
The generation is part of api repo `make generate` target. Check out the linked PR.
And we can’t generate it in build time. We need to fully control any possible changes in CRDs which can be introduced by newer kubefed. New CRD must go though PR reviews.

Reply:
ok, thanks for the clarifications @alexeykazakov